main.ts

```diff
@@ -445,27 +484,6 @@ export default class VimrcPlugin extends Plugin {
 
 	}
 
-	captureYankBuffer() {
-		if (this.yankToSystemClipboard) {
-			let currentBuffer = this.codeMirrorVimObject.getRegisterController().getRegister('yank').keyBuffer;

```

Comment:
It would probably make sense to always check something like:

```ts
const buffer = window.CodeMirrorAdapter.Vim.getRegisterController().getRegister('yank').keybuffer
```

to see whether the passed text via the event is equal to it and only then to copy it to the clipboard. That would have the effect that text only gets copied if it goes into the unnamed register, and not the Black hole or another register.